### DISCUSSION TITLE
TB v102.4.1 x64 on Win 10 - Authentication failure :: console log, solution ?

### DISCUSSION THREAD

**Discussion by Yachter** [kewisch/gdata-provider#526] [Support] [answered]
Calendar: [calGoogleSession] Authentication failure: {"error":"\n\u000finvalid_request\u0012Þ\u0001\nYou can't sign in to this app because it doesn't comply with Google's OAuth 2.0 policy for keeping apps secure.\n\nYou can let the app developer know that this app doesn't comply with one or more Google validation rules.\n  \u001a3https://support.google.com/accounts/answer/12379384 �\u0003*.\n\fredirect_uri\u0012\u001eurn:ietf:wg:oauth:2.0:oob:auto2£\u0002\b\u0001\u0012Í\u0001The out-of-band (OOB) flow has been blocked in order to keep users secure. Follow the Out-of-Band (OOB) flow Migration Guide linked in the developer docs below to migrate your app to an alternative method.\u001aOhttps://developers.google.com/identity/protocols/oauth2/resources/oob-migration","details":"\n\u000finvalid_request\u0012Þ\u0001\nYou can't sign in to this app because it doesn't comply with Google's OAuth 2.0 policy for keeping apps secure.\n\nYou can let the app developer know that this app doesn't comply with one or more Google validation rules.\n  \u001a3https://support.google.com/accounts/answer/12379384 �\u0003*.\n\fredirect_uri\u0012\u001eurn:ietf:wg:oauth:2.0:oob:auto2£\u0002\b\u0001\u0012Í\u0001The out-of-band (OOB) flow has been blocked in order to keep users secure. Follow the Out-of-Band (OOB) flow Migration Guide linked in the developer docs below to migrate your app to an alternative method.\u001aOhttps://developers.google.com/identity/protocols/oauth2/resources/oob-migration"} gdataSession.jsm:309
[Exception... "
�invalid_request���
You can't sign in to this app because it doesn't comply with Google's OAuth 2.0 policy for keeping apps secure.

You can let the app developer know that this app doesn't comply with one or more Google validation rules.
  �3https://support.google.com/accounts/answer/12379384 ��*.
�redirect_uri��urn:ietf:wg:oauth:2.0:oob:auto2�������The out-of-band (OOB) flow has been blocked in order to keep users secure. Follow the Out-of-Band (OOB) flow Migration Guide linked in the developer docs below to migrate your app to an alternative method.�Ohttps://developers.google.com/identity/protocols/oauth2/resources/oob-migration"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gdata-provider/legacy/modules/gdataSession.jsm :: login/authFailed< :: line 311"  data: no] 2 gdataSession.jsm:311:25
Unknown localization message errors.
�invalid_request���
you can't sign in to this app because it doesn't comply with google's oauth 2.0 policy for keeping apps secure.

you can let the app developer know that this app doesn't comply with one or more google validation rules.
  �3https://support.google.com/accounts/answer/12379384 ��*.
�redirect_uri��urn:ietf:wg:oauth:2.0:oob:auto2�������the out-of-band (oob) flow has been blocked in order to keep users secure. follow the out-of-band (oob) flow migration guide linked in the developer docs below to migrate your app to an alternative method.�ohttps://developers.google.com/identity/protocols/oauth2/resources/oob-migration ExtensionCommon.jsm:2070
Content Security Policy: Ignoring “'unsafe-inline'” within script-src or style-src: nonce-source or hash-source specified
Content Security Policy: Couldn’t process unknown directive ‘require-trusted-types-for’
Layout was forced before the page was fully loaded. If stylesheets are not yet loaded this may cause a flash of unstyled content. error:683:75
Cookie “GEM” does not have a proper “SameSite” attribute value. Soon, cookies without the “SameSite” attribute or with an invalid value will be treated as “Lax”. This means that the cookie will no longer be sent in third-party contexts. If your application depends on this cookie being available in such contexts, please add the “SameSite=None“ attribute to it. To know more about the “SameSite“ attribute, read https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite error
downloadable font: download failed (font-family: "Roboto" style:normal weight:500 stretch:100 src index:0): status=2152398850 source: https://fonts.gstatic.com/s/roboto/v18/KFOlCnqEu92Fr1MmEU9fABc4EsA.woff2
downloadable font: download failed (font-family: "Google Sans" style:normal weight:500 stretch:100 src index:0): status=2152398850 source: https://fonts.gstatic.com/s/googlesans/v14/4UabrENHsxJlGDuGo1OIlLU94YtzCwY.woff2
downloadable font: download failed (font-family: "Google Sans" style:normal weight:500 stretch:100 src index:0): status=2152398850 source: https://fonts.gstatic.com/s/googlesans/v14/4UabrENHsxJlGDuGo1OIlLU94Yt3CwZ-Pw.woff2
downloadable font: download failed (font-family: "Roboto" style:normal weight:400 stretch:100 src index:0): status=2152398850 source: https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu5mxKOzY.woff2
downloadable font: download failed (font-family: "Google Sans" style:normal weight:400 stretch:100 src index:0): status=2152398850 source: https://fonts.gstatic.com/s/googlesans/v14/4UaGrENHsxJlGDuGo1OIlL3Owp4.woff2
downloadable font: download failed (font-family: "Roboto" style:normal weight:400 stretch:100 src index:0): status=2152398850 source: https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu4mxK.woff2
downloadable font: download failed (font-family: "Roboto" style:normal weight:400 stretch:100 src index:0): status=2152398850 source: https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu7GxKOzY.woff2
downloadable font: download failed (font-family: "Google Sans" style:normal weight:400 stretch:100 src index:0): status=2152398850 source: https://fonts.gstatic.com/s/googlesans/v14/4UaGrENHsxJlGDuGo1OIlL3Kwp5MKg.woff2
downloadable font: download failed (font-family: "Roboto" style:normal weight:400 stretch:100 src index:0): status=2152398850 source: https://fonts.gstatic.com/s/roboto/v18/KFOmCnqEu92Fr1Mu4WxKOzY.woff2
downloadable font: download failed (font-family: "Roboto" style:normal weight:500 stretch:100 src index:0): status=2152398850 source: https://fonts.gstatic.com/s/roboto/v18/KFOlCnqEu92Fr1MmEU9fBBc4.woff2

**Comment by woody0130** [comment]
Same issue for me.  What to do ???

**Comment by xjonx1** [comment]
The app has to be updated. The developer has been contacted. The problem is the app needs googles new OAuth 2.0.

**Comment by xjonx1** [comment, marked as answer]
There was an issue with the add-on and is now fixed by the developer. Details here: https://github.com/kewisch/gdata-provider/discussions/519#discussioncomment-4147395
